src/transpiling/mod.rs

```diff
@@ -303,6 +310,16 @@ pub fn fold_program(
       ),
       options.transform_jsx
     ),
+    Optional::new(
+      as_folder(jsx_precompile::JsxPrecompile::new(
+        options.jsx_import_source.clone().unwrap_or_default(),
+        options.jsx_automatic,
```

Comment:
The `jsx_automatic` flag is not a good indicator for development mode or not. Typically the dev mode is enabled by setting a different transform. Like `react-jsxdev` vs `react-jsx` in the case of TypeScript https://www.typescriptlang.org/docs/handbook/jsx.html#basic-usage .

Reply:
It's a typo, it was supposed to be `jsx_development`.